internal/controller/crd_controller.go

```diff
@@ -68,6 +70,12 @@ func (r *CrdRegisterControllerReconciler) Reconcile(ctx context.Context, req ctr
 			continue
 		}
 
+		if crd.Labels != nil {
+			if _, ok := crd.Labels[common.ManagedByLabel]; !ok {
```

Comment:
shouldn't it be `ok` not `!ok` to check if it is managed by the agent?

Also we can probably move it out of the loop since watchers do not care about version and ignore it anyway

Reply:
i think what it should actually do is build the full set in the loop, will add that